src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -15,6 +16,19 @@ public class CSharpCodeWriter : CodeWriter
     {
         private const string InstanceMethodFormat = "{0}.{1}";
 
+        private static readonly char[] CStyleStringLiteralEscapeChars = new char[]
+        {
+            '\r',
+            '\t',
+            '\"',
+            '\'',
+            '\\',
+            '\0',
+            '\n',
+            '\u2028',
+            '\u2029',
+        };
```

Comment:
NVM. Only other occurences of `'\u2028'` are later in this file and in `ParserHelpers.IsNewLine()`.